downloaded/installLisk.sh

```diff
@@ -101,17 +101,21 @@ download_lisk() {
 		return
 	fi
 
-	LISK_VERSION=lisk-$UNAME.tar.gz
+	if [[ ! $LISK_VERSION_NUMBER ]]; then
+		LISK_VERSION_NUMBER=$(curl -s "https://downloads.lisk.io/lisk/$RELEASE/latest.txt")
+	fi
+
+	LISK_VERSION=lisk-$LISK_VERSION_NUMBER-$UNAME.tar.gz
 
 	LISK_DIR=${LISK_VERSION%.tar.gz}
 
 	rm -f "$LISK_VERSION" "$LISK_VERSION".SHA256 &> /dev/null
 
 	echo -e "\\nDownloading current Lisk binaries: ""$LISK_VERSION"
 
-	curl --progress-bar -o "$LISK_VERSION" "https://downloads.lisk.io/lisk/$RELEASE/$LISK_VERSION"
+	curl --progress-bar -o "$LISK_VERSION" "https://downloads.lisk.io/lisk/$RELEASE/$LISK_VERSION_NUMBER/$LISK_VERSION"
```

Comment:
Same as above: we should fail with an understandable error message if the file cannot be downloaded.